src/__e2e__/publishRegistry.test.ts

```diff
@@ -31,75 +33,9 @@ describe('publish command (registry)', () => {
       repositoryFactory.cleanUp();
       repositoryFactory = undefined;
     }
-    if (spy) {
-      spy.mockRestore();
-      spy = undefined;
-    }
-  });
-
-  it('will perform retries', async () => {
```

Comment:
This test is still here, but I moved it to the end--I think it makes more sense to test normal/successful publishing scenarios first